CRM/Contact/Form/Task/SaveSearch.php

```diff
@@ -242,4 +240,17 @@ public function postProcess() {
     }
   }
 
+  /**
+   * Set form defaults.
+   *
+   * return array
+   */
+  public function setDefaultValues() {
+    $defaults = array();
+    if (empty($defaults['parents'])) {
+      $defaults['parents'] = CRM_Core_BAO_Domain::getGroupId();
```

Comment:
I just checked & this behaviour is also in CRM_Group_Form_Edit - the multisite extension overrides it.

- would be good to get this merged